nexus/tests/integration_tests/schema.rs

```diff
@@ -168,13 +168,14 @@ async fn query_crdb_schema_version(crdb: &CockroachInstance) -> String {
 //
 // Note that for the purposes of schema comparisons, we don't care about parsing
 // the contents of the database, merely the schema and equality of contained data.
-#[derive(Eq, PartialEq, Clone, Debug)]
```

Comment:
I had to get rid of `Eq` to make this work because floats don't impl `Eq`. Fortunately it seems our equality checks only need `PartialEq` anyway.